integreat_cms/release_notes/current/unreleased/3436.yml

```diff
@@ -0,0 +1,2 @@
+en: Fix german translation of recurring
+de: Ändere die deutsche Übersetzung von wiederkehrend zu wiederholdend
```

Comment:
small typo in this release note: wiederholdend instead of wiederholend

Reply:
Good catch! Thanks :)